packages/client/src/client/OpenFin/apps/Launcher/NlpSuggestions/executions/RfqExecution/RfqExecutionWorkflow.tsx

```diff
@@ -22,10 +23,20 @@ const ConfirmContent = ({
   direction,
   notional,
   symbol,
+  maturity,
 }: RfqNlpExecutionDataReady["payload"]["requestData"]) => {
   useMoveNextOnEnter(onNext)
   const directionStr = direction === Direction.Buy ? "buying" : "selling"
   const notionalStr = formatNumber(notional * 1000)
+  const creditInstrument = creditInstruments$
+    .getValue()
+    .filter((bond) => bond.ticker === symbol)
+
+  const selectedMaturity =
+    maturity === creditInstrument[0].maturity.slice(0, 4) ||
+    maturity === creditInstrument[1].maturity.slice(0, 4)
```

Comment:
Should this be written to accommodate for cases where there are only one maturity or more than 2 maturities per instrument?

Reply:
I know currently we always have two maturities per instrument, but we could potentially add more maturities/instruments in the future.

Reply:
yeah, this is too brittle .. we are in control of the test data, so we have 2 of each currently, but someone could helpfully amend that data at any point so we have more instruments to play with ..